zookeeper/src/main/java/com/linecorp/armeria/client/zookeeper/StoreType.java

```diff
@@ -0,0 +1,8 @@
+package com.linecorp.armeria.client.zookeeper;
+
+/**
+ * Value store type. As a node value or as a node's all Children.
+ */
+public enum StoreType {
+    NodeChild,NodeValue
```

Comment:
- Missing Javadoc. How about adding some explanation about the layout of data here in detail?
- How about: `NodeChild` -> `IN_CHILD_NODES`
- How about: `NodeValue` -> `IN_NODE_VALUE`